lib/order.rb

```diff
@@ -0,0 +1,87 @@
+require_relative '../lib/menu.rb'
+require 'date'
+require 'twilio-ruby'
+
+class Order < Menu
+  attr_reader :basket, :updated_basket
+
+  def initialize
+    @basket = []
+    @updated_basket = {}
+  end
+
+  def order_dish(quantity = 1, dish)
+    menu_item = lookup_item(dish)
+    puts quantity.to_s + ' ' + dish + '(s) added to your basket.'
+    while quantity > 0
+      @basket << menu_item
+      quantity -= 1
+    end
+
+  end
+
+  def lookup_item(dish)
+    fail 'Please enter an item from the menu.' unless @@dishes.key?(dish)
+    chosen_dish = @@dishes.select do |item, _price|
+      item == dish
+    end
+    chosen_dish
+  end
+
+  def create_basket_items
```

Comment:
Could the encapsulation of the `Order` class be improved? One way to improve encapsulation is to try to make as many methods as possible private and then look to see if there are any responsibilities in the collective private methods that could be broken out.